src/atomate2/settings.py

```diff
@@ -84,6 +84,11 @@ class Atomate2Settings(BaseSettings):
         description="Whether to run the Bader program when parsing VASP calculations."
         "Requires the bader executable to be on the path.",
     )
+    VASP_RUN_DDEC6: bool = Field(
```

Comment:
It also requires an environment variable `DDEC6_ATOMIC_DENSITIES_DIR`. I mean, by default it assumes local directory but it's unlikely the user will copy the atomic densities to the local dir if not directed.

Reply:
Yeah, we were discussing over at https://github.com/materialsproject/emmet/pull/869#issuecomment-1773481631. I think it would be best not to bother the user with this and just handle it automatically.

Reply:
I see. Just double-check the licensing that this would be permitted, assuming you mean that you'll ship it.

Reply:
Not ship in the `pymatgen` package since it would burden many non-users of `chargemol` to download those files. But once they're in the repo, we have a stable `raw.github.com` URL from where we could `wget` them.

Reply:
Ah, that's smart. But one thing to note is that not all compute nodes have access to the network, so you might want to have a fallback option.

Reply:
Yes, fallback option for no-network situations is on the TODO list.

Reply:
Oh yes, please. Most computing centers in Europe don't have any outside connection on the compute nodes.

Reply:
Or at Princeton... 😉